lib/args.js

```diff
@@ -13,17 +13,17 @@ function urlWithArgs (urlOrFile, args) {
   args = encode(args)
 
   var u
+  var urlData = url.parse(urlOrFile)
   if (urlOrFile.indexOf('http') === 0) {
```

Comment:
Is `url.protocol` something might be useful here? https://nodejs.org/api/url.html#url_url_protocol

Reply:
that is probably for the best.  @L3O do you want to change this before I merge it?

Reply:
Done.